src/Orchard.Web/Modules/Orchard.Azure/Orchard.Azure.csproj

```diff
@@ -144,12 +144,6 @@
     <Compile Include="Services\Environment\Configuration\DefaultPlatformConfigurationAccessor.cs" />
     <Compile Include="Services\Environment\Configuration\IPlatformConfigurationAccessor.cs" />
     <Compile Include="Services\Environment\AzureApplicationEnvironment.cs" />
-    <None Include="Microsoft.WindowsAzure.Caching\ClientPerfCountersInstaller.exe">
```

Comment:
Was this to not copy the file? Is the file still around?

Reply:
The files ClientPerfCountersInstaller.exe and PerformanceCounters.xml were never been in the csproj and neither in git before #8444.
the command "Update-Package -reinstall" added them to csproj; because of the high number of projects I did not notice the addition. So I removed them in this PR.